src/components/ImageView/index.js

```diff
@@ -35,7 +35,7 @@ class ImageView extends PureComponent {
         this.trackPointerPosition = this.trackPointerPosition.bind(this);
 
         this.state = {
-            isLoading: false,
+            isLoading: true,
```

Comment:
I want to change this line because it can improve the performance a bit. isLoading is always set to true then false when image is loaded.

Reply:
Yeah, I agree, it should be set to true initially

Reply:
Is this ready for testing?